src/matrixrtc/MatrixRTCSession.ts

```diff
@@ -117,7 +132,7 @@ export class MatrixRTCSession extends TypedEventEmitter<MatrixRTCSessionEvent, M
     }
 
     private constructor(
-        private readonly client: MatrixClient,
+        public readonly client: MatrixClient,
```

Comment:
aaaargh, I was able to make this private in the refactor, can't we keep it this way? :)